package.json

```diff
@@ -59,7 +60,9 @@
   },
   "devDependencies": {
     "@types/ember": "2.8.8",
+    "@types/ember-qunit": "^3.0.1",
```

Comment:
My guess is that this is referencing Ember’s types, and that’s how we’re ending up with the dupes. Probably need to throw a `resolutions` field in here. UGH.

Reply:
Confirmed, that’s the problem.

I *think* we’re okay if we add the resolutions field b/c it’s only for `devDependencies`… but I’m honestly not 100% confident in the semantics for yarn there. I’ll push a change with that and then we can ask for people to experiment with it in their own apps and addons.